src/app/src/reducers/FiltersReducer.js

```diff
@@ -16,6 +16,8 @@ import {
     completeFetchCountryOptions,
 } from '../actions/filterOptions';
 
+import { completeSubmitLogOut } from '../actions/auth';
```

Comment:
Ah, great work. I see some other reducers also import and use this, but some others don't:

```bash
$ ag -L completeSubmitLogOut src/app/src/reducers/

src/app/src/reducers/index.js
src/app/src/reducers/UIReducer.js
src/app/src/reducers/UploadReducer.js
src/app/src/reducers/FilterOptionsReducer.js
src/app/src/reducers/FacilitiesReducer.js
src/app/src/reducers/OARMapReducer.js
```

Is it alright for them not to have it? Should this be something "higher up" that wraps the other reducers so we don't have to keep doing it manually?

Reply:
Yeah, those other reducers don't store any user-related data so I think they're good. However, I think the `FacilitiesReducer` should be dumped because it needs to stay harmonized with the set of selected filters in the FiltersReducer.